build.sbt

```diff
@@ -0,0 +1,44 @@
+import sbt.Keys._
+
+name := "scalajs-java-time"
+
+version := "0.0.1-SNAPSHOT"
+
+organization := "org.scala-js"
+
+scalaVersion := "2.11.7"
+
+scalacOptions ++= Seq("-deprecation", "-feature", "-Xfatal-warnings")
+
+homepage := Some(url("http://scala-js.org/"))
+
+licenses += ("MIT", url("http://opensource.org/licenses/mit-license.php"))
+
+testOptions += Tests.Argument(
+    TestFramework("com.novocode.junit.JUnitFramework"), "-v", "-a")
+
+val commonSettings = Seq(testOptions += Tests.Argument(TestFrameworks.JUnit, "-v", "-a"))
+
+lazy val javaTime: Project = Project(
```

Comment:
And if you call this
```scala
lazy val `scalajs-java-time` = ...
```
you don't need to set the `name` (apparently this is common practice in sbt builds, nowadays)